build-tools/packages/build-cli/src/commands/generate/changeset.ts

```diff
@@ -119,11 +124,32 @@ export default class GenerateChangesetCommand extends BaseCommand<typeof Generat
 		const remote = await repo.getRemote(context.originRemotePartialUrl);
 
 		if (remote === undefined) {
-			// Logs and exits
 			this.error(`Can't find a remote with ${context.originRemotePartialUrl}`, { exit: 1 });
 		}
 		this.log(`Remote for ${context.originRemotePartialUrl} is: ${chalk.bold(remote)}`);
 
+		// If the branch flag was passed explicitly, we don't want to prompt the user to select one. We can't check for
+		// undefined because there's a default value for the flag.
+		const usedBranchFlag = this.argv.includes("--branch") || this.argv.includes("-b");
```

Comment:
`argv` is an array, not a string, right? So the second check won't get confused by a hypothetical `--build` flag?

Reply:
Correct, it's an array.